### PR TITLE
Mastodon daily ditado tooter

### PR DESCRIPTION
**Pull request by henriquetguedes**
implements a 9:30am scheduler that toots a random ditado, provided an access token is present; mute otherwise.
-added masto, dotenv, and node-schedule dependencies

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,3 +33,16 @@ server.use(router)
 server.listen(PORT, () => {
   console.log(`Server running on port ${PORT}`)
 })
+
+async function tooter(msg) {
```
Comment (antonioalmeida) [inline index.js:37]: this can be in a separate file, e.g., `tooter.js`, ideally with a couple of tests (tests can be done later)

File: index.js [hunk 2]
```diff
@@ -29,3 +33,16 @@ server.use(router)
...
+  // URL and access token for a specific Mastodon account should be defined as env variables
+  const masto = await login({ url: process.env.MAST_URL, accessToken: process.env.MAST_TOKEN })
+  await masto.statuses.create({ status: msg, visibility: 'public' })
+}
+
+if (process.env.MAST_TOKEN) { // assumes a Mastodon URL is also set and takes it as a sufficient condition for running
```
Comment (antonioalmeida) [inline index.js:43]: `MAST_TOKEN` is not the mastodon host, right? the comment is misleading

**Review by antonioalmeida — CHANGES_REQUESTED** [review]
Looking good

Please also add a `.env.example` file with sample value for the environment variables. ([why?](https://stackoverflow.com/questions/52084447/what-is-the-need-for-env-and-env-example-files-in-laravel))

**Comment by henriquetguedes** [comment]
- `.env.example` file added
- `MAST_URL` comment clarified
- `tooter.js` created

**Pull request merged**